LastWork(newWork)/css/p_evaluations.css

```diff
@@ -0,0 +1,81 @@
+.image{
+	width: 150px;
+	height: 30px;
+	margin-left: 50px;
+}
+
+.header{
+	width: 100%;
+	height: 55px;
+	border-bottom: 1px solid;
+	display: inline-flex;
+	align-items: center;
+	box-shadow: 0px 3px 10px grey;
+}
+
+.header_div{
+	width: 1200px;
+	width: 1200px;
+	margin: 0 auto;
+}
+
+.body_section{
+	width: 1200px;
+	height: 700px;
+	margin: 0 auto;
+}
+
+.eval_button{
+	border-bottom: 10px solid;
+	border-bottom-color: #03a8f3;
+	text-decoration: none;
+	padding: 0px 50px;
+	float:left;
+	margin-bottom: 0px;
+}
+
+.neweval_button{
+	text-decoration: none;
+	padding: 0px 50px;
+	float: left;
+	margin-bottom: 0px;
+}
+
+.logout{
+	float: right;
+	text-decoration: none;
+	margin-right: 50px;
+}
+
+.table
+{
+	width: 100%;
+	margin-top: 150px;
+	font-family: arial, sans-serif;
+	border-collapse: collapse;
+}
+
+.table td, th 
```

Comment:
Just making sure that you are indeed targeting only `td` elements within an ancestor with the `table` class, along with _all_ `th` elements, regardless of whether or not any of their ancestors have a `table` class.